domain-models-runtime/src/main/java/org/folio/rest/impl/TenantAPI.java

```diff
@@ -96,12 +96,15 @@ public void deleteTenant(Map<String, String> headers,
             postgresClient(context).runSQLFile(sqlFile, true,
                 reply -> {
                   try {
+                    // close clients because they still use the old oid of the dropped schema/role name,
+                    // they will fail when the same schema/role is recreated with a new oid.
+                    PostgresClient.closeAllClients();
```

Comment:
Why close connections for all tenants? Won't the current tenant suffice?

Reply:
Good catch! Fixed.